fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -1556,15 +1554,11 @@ private void handleShowWorkGroup() {
 
     private void handleShowCatalogs() {
         ShowCatalogsStmt showCatalogsStmt = (ShowCatalogsStmt) stmt;
-        ConcurrentHashMap<String, Catalog> catalogs = GlobalStateMgr.getCurrentState().getCatalogMgr().getCatalogs();
-        List<List<String>> rows = Lists.newArrayList();
-        for (Map.Entry<String, Catalog> entry : catalogs.entrySet()) {
-            List<String> catalog = Lists.newArrayList();
-            catalog.add(entry.getKey());
-            Catalog value = entry.getValue();
-            catalog.add(((ExternalCatalog) value).getType());
-            catalog.add(value.getComment());
-            rows.add(catalog);
+        List<List<String>> rows = new ArrayList<>();
+        rows.add(Arrays.asList("default", "default", "internal catalog"));
```

Comment:
how about moving this into CatalogMgr? cc @stephen-shelby

Reply:
yes, plz ref ResourceMgr.ResourceProcNode